images/bootstrap/Dockerfile

```diff
@@ -33,6 +33,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     file \
     git \
+    golang \
```

Comment:
this should not have merged.
go is already installed in kubekins, and we use the correct version for the kubernetes branch, NOT whatever is in debian (which is probably super old)